staging/publishing/rules.yaml

```diff
@@ -622,3 +622,38 @@ rules:
       dir: staging/src/k8s.io/cri-api
     name: release-1.15
     go: 1.12.5
+- destination: kubectl
+  library: true
+  branches:
+  - source:
+      branch: master
+      dir: staging/src/k8s.io/kubectl
+    name: master
+    dependencies:
+    - repository: api
```

Comment:
Since the repo is empty right now, do you want to add these dependencies when you start populating the repo?

Having them here won't harm but in case you want to keep them here, can you also add kubectl to https://github.com/kubernetes/kubernetes/blob/master/staging/publishing/import-restrictions.yaml? :)

Reply:
I added the allowed imports into `import-restrictions.yaml`